pkgs/tools/misc/nvimpager/default.nix

```
@@ -32,12 +32,16 @@ stdenv.mkDerivation rec {
   checkInputs = [ lua51Packages.busted util-linux neovim ];
```

You need to add `script` for Darwin here too, `unixtools.script` will do that (see [here](https://github.com/NixOS/nixpkgs/blob/9c16fe5b873a2919ad324aeb75fa5aa8f5da0c2c/pkgs/top-level/unixtools.nix#L142)).

```suggestion
  checkInputs = [ lua51Packages.busted unixtools.script neovim ];
```

---

Done. Thanks.